docs/building/windows-instructions.md

```diff
@@ -56,8 +56,8 @@ You can use the Developer Command Prompt, Powershell or work in any regular cmd.
 From a (non-admin) Command Prompt window:
 
 - `build.cmd` - builds the assemblies
-- `build.cmd -runTests` - called after a normal "build.cmd" will run all tests
-- `build.cmd -buildPackages` called after a normal “build.cmd” will create the NuGet packages with the assemblies in “bin"
+- `build.cmd -test` - called after a normal "build.cmd" will run all tests
```

Comment:
(appended to unrelated code to allowed for threaded comments)\

As requested by @harishsk on the chat, can you please add instructions about how to run specific tests on the CI?

Since it's about running tests on the CI, and not "building locally" (which is what these docs on this PR are about), maybe it would be better to add a new markdown file with the instructions for running specific tests on the CI.

The instructions would be the same as always... set "runSpecific:true" and "innerLoop:false", and [TestCategory("RunSpecificTest")], as shown on this PR:
https://github.com/dotnet/machinelearning/pull/5539/commits/2fb5f8cfcd2a81f27bc22ac6749f1ce2045e925b

thanks!

Reply:
You can also add it to the [developer-guide](https://github.com/dotnet/machinelearning/blob/master/docs/project-docs/developer-guide.md)

While you are at it, can you please review the developer-guide and make sure it is up to date?

Reply:
Good point, I have added the instructions to the developer guide, and updated the developer guide per the Arcade SDK update.